crashlog/crashutils.c

```diff
@@ -178,6 +180,33 @@ void do_last_kmsg_copy(char *dir) {
         snprintf(destion, sizeof(destion), "%s/%s", dir, FTRACE_RAMOOPS_FILE);
         do_copy_tail(FTRACE_RAMOOPS, destion, MAXFILESIZE);
     }
+    source = find_system_last_kmsg();
+    if (source) {
+        snprintf(destion, sizeof(destion), "%s/%s", dir, source);
+        snprintf(sourcepath, sizeof(sourcepath), "%s/%s", DROPBOX_DIR, source);
+        do_copy_tail(sourcepath, destion, MAXFILESIZE);
+        free(source);
+    }
+}
+
+// Find system last kmsg from dropbox
+char *find_system_last_kmsg() {
```

Comment:
Suggest to declare this function as static function if not called in other places.